phalcon/debug.zep

```diff
@@ -331,6 +345,37 @@ class Debug
 		}
 	}
 
+	/**
+	 * Sets if files the exception's backtrace must be showed
+	 */
+	public function setBlacklist(array blacklist) -> <Debug>
+	{
+		var area, result, subArray, value;
+
+		let area     = Arr::get(blacklist, "request", []),
+			subArray = [],
+			result   = [];
+
+		for value in area {
+			let value           = strtolower(value),
+				subArray[value] = 1;
+		}
+
+		let result["request"] = subArray,
+			area              = Arr::get(blacklist, "server", []),
+			subArray          = [];
```

Comment:
Why so many spaces between variables and `=` in this method?

Reply:
Space driven development :)

Reply:
The tabs on the left side throw the whole thing off. Everything is actually aligned properly (equal signs and vars)